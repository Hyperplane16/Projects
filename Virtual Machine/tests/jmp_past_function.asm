FUNC LABEL 0
    MOV REG 0 VAL 25
    MOV REG 1 VAL 30
    MOV REG 7 VAL 5
    MOV REG 2 VAL 35
    MOV REG 3 VAL 40
    RET
