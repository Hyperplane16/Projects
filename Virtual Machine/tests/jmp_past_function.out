PC references non-existent instruction in the function scope.
1